promtimer/promtimer.py

```diff
@@ -338,19 +355,40 @@ def main():
         min_time = 'now-30m'
         max_time = 'now'
         refresh = args.refresh or '5s'
+    elif args.backup_archive_path:
+        backup_archive_mode = True
+        stats_sources, min_time, max_time, refresh = backupstats.handle_backup_archive_mode(
+            args.backup_archive_path,
+            prometheus_base_port
+        )
+    else:
+        stats_sources = cbstats.CBCollect.get_stats_sources(prometheus_base_port)
+        if not stats_sources:
+            sys.exit(1)
+        times = cbstats.CBCollect.compute_min_and_max_times(stats_sources)
+        min_time = datetime.datetime.fromtimestamp(times[0]).isoformat()
+        max_time = datetime.datetime.fromtimestamp(times[1]).isoformat()
+        refresh = ''
 
-    if not args.buckets:
+
```

Comment:
Too many blank lines before the if statement (should just be one). Again see PEP standard.